x-pack/plugins/security_solution/common/endpoint/types/index.ts

```diff
@@ -1095,3 +1096,14 @@ export interface HostPolicyResponse {
 export interface GetHostPolicyResponse {
   policy_response: HostPolicyResponse;
 }
+
+/**
+ * REST API response for retrieving agent summary
+ */
+export interface GetAgentSummaryResponse {
+  summary_response: {
+    package: string;
+    policy_id?: string;
+    versions_count: JsonObject;
```

Comment:
Why support such a loose type for the value here? Should this instead be: `Record<string, number>`?